### PR TITLE
TOPS-867 - add required roles for dataform

### PR DESCRIPTION
**Pull request by kthhrv**
tog-staging-dt-prc-0@tog-staging-dt-prc.iam.gserviceaccount.com
- bigquery.admin on processing and landing

service-714489453670@gcp-sa-dataform.iam.gserviceaccount.com (https://cloud.google.com/dataform/docs/required-access)
- bigquery.dataViewer on landing
- bigquery.dataEditor on processing and curated
- bigquery.jobUser on processing

### CODE REVIEW

File: blueprints/data-solutions/data-platform-minimal/01-landing.tf [hunk 1]
```diff
@@ -19,6 +19,7 @@ locals {
     "roles/storage.objectCreator" = [module.land-sa-0.iam_email]
     "roles/storage.objectViewer"  = [module.processing-sa-cmp-0.iam_email]
     "roles/storage.objectAdmin"   = [module.processing-sa-0.iam_email]
+    "roles/bigquery.dataViewer"   = ["serviceAccount:${module.processing-project.service_accounts.robots.dataform}"]
```
Comment (Munir93) [inline blueprints/data-solutions/data-platform-minimal/01-landing.tf:23]: We need [tog-staging-dt-prc-0@tog-staging-dt-prc.iam.gserviceaccount.com] to be bq admin on landing

Comment (kthhrv) [reply blueprints/data-solutions/data-platform-minimal/01-landing.tf:23]: ok adding

Comment (kthhrv) [reply blueprints/data-solutions/data-platform-minimal/01-landing.tf:23]: but not curated ?

Comment (Munir93) [reply blueprints/data-solutions/data-platform-minimal/01-landing.tf:23]: Hmm I dont think so as its dataform that will be talking to curated and that has its own service account. Which reminds me Dataform account will need Admin on curated

Comment (Munir93) [reply blueprints/data-solutions/data-platform-minimal/01-landing.tf:23]: Ignore that dataform only needs editor on curated

**Review by Munir93 — APPROVED** [review]

**Pull request merged**